src/util/accountUtils.ts

```diff
@@ -252,60 +259,36 @@ const linkAddressToNamespace = (isMultisig :boolean,cosigners: [], multisigAccou
-      let cosignerPrivateKey = WalletUtils.decryptPrivateKey(new Password(walletPassword), firstCosigner.encrypted, firstCosigner.iv);
-      coSigner.splice(0,1)
-      let account = Account.createFromPrivateKey(cosignerPrivateKey,networkType) 
-      
-      aggregateTransaction = AggregateTransaction.createComplete(
-        Deadline.create(), 
-        innerTx,
-        networkType,
-        []
-      )
-      let signedAggregateCompleteTransaction = account.signTransactionWithCosignatories(
-        aggregateTransaction,coSigner,Hash
-      )
-      announceTransaction(signedAggregateCompleteTransaction)
-      signedTransaction = signedAggregateCompleteTransaction
-    }else{ //aggregate bonded
-      let newBuildTx = new BuildTransactions(networkType);
-      const aggreateBondedTx = newBuildTx.aggregateBonded(innerTx);
-      let transactions :{signedAggregateBondedTransaction:SignedTransaction,lockFundsTransactionSigned:SignedTransaction}[] = [];
-      coSigner.forEach(signer=>{
-        const signedAggregateBondedTransaction = signer.sign(aggreateBondedTx, Hash);
-      
-        let hashLockTx = newBuildTx.hashLock(
-          Helper.createAsset(networkState.currentNetworkProfile.network.currency.assetId, 10000000), 
-          Helper.createUint64FromNumber(200),
-          signedAggregateBondedTransaction 
-        );
-
-      const lockFundsTransactionSigned = signer.sign(hashLockTx, Hash);
-
-      transactions.push({ signedAggregateBondedTransaction: signedAggregateBondedTransaction, lockFundsTransactionSigned: lockFundsTransactionSigned });
-      })
-      signedTransaction = transactions[transactions.length-1].signedAggregateBondedTransaction
-      for (const transaction of transactions) {
-        multiSigAnnouce(transaction.signedAggregateBondedTransaction,transaction.lockFundsTransactionSigned)
-      }
-    }
+    
+    let txBuilder = AppState.buildTxn;
+    const aggreateBondedTx = txBuilder.aggregateBonded(innerTx);
+    const firstSigner = cosignerAcc[0]
+    cosignerAcc.splice(0,1)
+    const signedAggregateBondedTransaction = firstSigner.signTransactionWithCosignatories(aggreateBondedTx,cosignerAcc,Hash);
+    signedTransaction = signedAggregateBondedTransaction
+    const nativeTokenNamespace = networkState.currentNetworkProfile.network.currency.namespace;
+
+    const lockingAtomicFee = networkState.currentNetworkProfileConfig.lockedFundsPerAggregate ?? 0;
+
+    const lockFundsTransaction = txBuilder.hashLock(  
+      new Mosaic(new NamespaceId(nativeTokenNamespace), UInt64.fromUint(lockingAtomicFee)),
+      UInt64.fromUint(1000),
+      signedAggregateBondedTransaction,
+    )
+    const lockFundsTransactionSigned = firstSigner.sign(lockFundsTransaction , Hash);
+    signedTransaction = signedAggregateBondedTransaction
+    multiSigAnnouce(signedAggregateBondedTransaction,lockFundsTransactionSigned)
```

Comment:
maybe create a method in a class and import it ? why transaction processing are here anyway ?